page/docs/guide/json-rpc-api.md

```diff
@@ -31,3 +31,31 @@ Response:
 Methods currently not supported:
 
 - `starknet_protocolVersion` - will be removed in a future version of the specification
+
+Methods that require a `block_id` only support ids of the `latest` or `pending` block:
+
+```json
```

Comment:
`json` doesn't support comments, so if you could change the language to `js` or `ts`.